### ISSUE TITLE
Mountain Lions Stealth  Alterac Mountains

### ISSUE THREAD

**Issue by Resistofc** [Wall-core/Everlook-Bugtracker#384]
Normal Mountain Lions (Not Hulking) in Alterac Mountains are stealthed but they should not be.

**Comment by Tessablanc** [comment]
This is not correct. It isn't only Hulking lions that have stealth. In fact, if you tame a Mountain Lion, they actually come with the Prowl ability. Because they use it.

**Comment by Resistofc** [comment]
They never used that ability back then, on classic or on lights hope, when not tamed.

Holen Sie sich Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Tessablanc** [comment]
Every single cat that comes with Prowl is stealthed in the open world: https://www.wow-petopia.com/classic/abilities.php#prowl

There's no reason to think Mountain Lion is an exception to this. Why would it be? This is just another example of Classic having mistakes and incorrect data. Cats that have Prowl are stealthed. That's why they have Prowl. It would make no sense otherwise.

**Comment by Resistofc** [comment]
I played Vanilla back then and I know that they were not stealthed and Classic was right about it.
There are some annoying things like stealthed cats, that jump you every few seconds, you just don't forget lol.
Also you can watch Joanas speedleveling Videos, where you can see them not stealthed.
https://youtu.be/D9O903Wnmb4?t=7550

**Comment by Tessablanc** [comment]
Most likely blizz made a mistake with them being unstealthed originally and then fixed it in TBC. Clearly they're meant to be stealthed since they have Prowl. Why would a cat that is able to Prowl not be using that ability, ever? That's pure nonsense. Things are coded in this game in a way that is flavorful and sensical, not just arbitrarily some way "because Daddy said so."

Really this complaint about Mountain Lion stealth exposes a whiny, authoritarian, non-creative, borderline-autistic mentality where you think that the rubric for how things should be is "what Blizzard did," even when Blizzard clearly made a mistake. Every single cat that comes with Prowl is stealthed. If blizz forgot to do this in 2005, it was a mistake on their part. But instead of understanding this basic logic, you make an empty appeal to authority, as if a decision someone made is justified just because they made it. Mountain Lions have Prowl and therefore are meant to be stealthed, exactly like every other cat with Prowl. _Period._ Furthermore, the fact that they're in stealth fits the flavor of the part of Alterac they're in, which also has a bunch of rogue NPCs, many of whom are stealthed as well.

So you want to take away that interesting and obviously-intended flavor, and make the area blander and more boring, just because you think "that's what Daddy did" in 2005. Guess what? Daddy made a mistake! An unstealthed cat that has Prowl is an inferior and nonsensical design decision, and clearly an error, especially considering how it detracts from the flavor of the overall area. You want the game to be changed in a way that is totally boring and lame, and clearly objectively wrong, just because you think "Daddy did it." Again: Guess what? Daddy makes mistakes. And this was obviously one of them. The fact that pservers have rectified that mistake in the past decade is purely an improvement.

**Comment by Resistofc** [comment]
If it was because Blizzard made a mistake and fixed it in TBC, then it's a TBC thing and not Vanilla. This Vanilla private server aims to recreate the Vanilla gameplay experience and not the "how it was maybe intended to be because it's more logical this way", therefore only stuff that was actually present matters.

This has nothing to do with being whiny. They want us to report bugs and stuff that is off compared to the original game and it's just what i did here.

If you want a private server that trys to interpret blizzard's intentions from 2005, that didn't make it into the game, then you are clearly wrong here.

I gave you troll already more attention than I should.

You are just triggered as fuck that I'm right and were able to prove this. _Period_

**Comment by Tessablanc** [comment]
Again you don't have any argument whatsoever. You just keep repeating "Daddy's mistake is good because Daddy made it" over and over, like an authoritarian baby. Private servers offer the opportunity to correct stupid mistakes that were made in the past, rather than repeating them forever just because they existed at some point. And your claim that "This Vanilla private server aims to recreate the Vanilla gameplay experience," as if you are a dev here, is empty and fake. Blizzard's intention in vanilla was for Mountain Lions to be stealthed. They fucked up. Now we will correct their mistake, instead of wallowing in completely inane and pointless failure forever while pretending like doing so is some kind of accomplishment. Real life moves on, and intelligent people think about things and fix and improve them, while stupid braindead losers whine about "muh Daddy's broken Mountain Lions."

**Comment by Resistofc** [comment]
tl;dr

must be a troll
